jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -267,6 +267,8 @@ def _parse_hdulist(app, hdulist, file_name=None,
             app.add_data_to_viewer(spectrum_viewer_reference_name, data_label)
             app._jdaviz_helper._loaded_flux_cube = app.data_collection[data_label]
 
+        metadata['_parsed_before'] = True
```

Comment:
This feels a little hacky. At the tag-up on 2024-02-02, we have agreed to wait for specutils 2.0 effort to see if we would even need such workaround then.

Thanks!

Reply:
Agreed, Kyle had asked to me to create a Draft PR for this temporary solution to bring up to Cami when we re-discuss the topic of save/save-state/etc. sometime next week; so this PR was only created to not lose track of an unideal solution and to offer a temporary solution that would need to be reverted post specutils 2.0. We were going to let her decided whether or not to use it and revert or to wait for 2.0 in general